src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/ExecuteReconfigurationTaskOperationControllerTest.java

```diff
@@ -399,6 +407,132 @@ void should_execute_reconfigure_on_task_and_update_data_when_can_reconfigure_is_
+        taskResourcesBefore.forEach(task -> {
+            assertEquals(ASSIGNEE_USER, task.getAssignee());
+            assertEquals(CFTTaskState.ASSIGNED, task.getState());
+            assertNotNull(task.getReconfigureRequestTime());
+            assertEquals(OffsetDateTime.now().toLocalDate(), task.getReconfigureRequestTime().toLocalDate());
+            assertNull(task.getMinorPriority());
+            assertNull(task.getMajorPriority());
+            assertNull(task.getDescription());
+            assertNull(task.getCaseName());
+            assertEquals("765324", task.getLocation());
+            assertEquals("Taylor House", task.getLocationName());
+            assertNull(task.getCaseCategory());
+            assertNull(task.getWorkTypeResource());
+            assertNull(task.getRoleCategory());
+            assertEquals(OffsetDateTime.now().toLocalDate(), task.getPriorityDate().toLocalDate());
+            assertNull(task.getNextHearingDate());
+            assertNull(task.getNextHearingId());
+            assertNotNull(task.getDueDateTime());
+        });
+
+        when(dmnEvaluationService.evaluateTaskConfigurationDmn(
+            anyString(),
+            anyString(),
+            anyString(),
+            anyString())).thenReturn(configurationDmnResponse(true));
+        when(dmnEvaluationService.evaluateTaskPermissionsDmn(
+            anyString(),
+            anyString(),
+            anyString(),
+            anyString())).thenReturn(permissionsResponse());
+        when(cftQueryService.getTask(eq(taskResourcesBefore.get(0).getTaskId()), any(), anyList())).thenReturn(Optional.of(taskResourcesBefore.get(0)));
+        String secondTaskId = taskResourcesBefore.get(1).getTaskId();
+        //Throwing exception for second task while reassigning the task
+        when(cftQueryService.getTask(eq(secondTaskId), any(), anyList())).thenThrow(new RuntimeException("Exception occurred while reassigning the task"));
+        when(cftQueryService.getTask(eq(taskResourcesBefore.get(2).getTaskId()), any(), anyList())).thenReturn(Optional.of(taskResourcesBefore.get(2)));
+        mockMvc.perform(
+            post(ENDPOINT_BEING_TESTED)
+                .header(SERVICE_AUTHORIZATION, SERVICE_AUTHORIZATION_TOKEN)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .content(asJsonString(taskOperationRequest(
+                    EXECUTE_RECONFIGURE,
+                    executeTaskFilters(OffsetDateTime.now().minusSeconds(30L))
+                )))
+        ).andExpectAll(
+            status().is(HttpStatus.OK.value())
+        );
+        Thread.sleep(5000);
+        List<TaskResource> taskResourcesAfter = cftTaskDatabaseService.findByCaseIdOnly(caseIdToday);
+
+
+        taskResourcesAfter
+            .stream().filter(taskResource -> !taskResource.getTaskId().equals(secondTaskId))
+            .forEach(task -> {
+                assertEquals(1, task.getMinorPriority());
```

Comment:
can we add 2 more assertions to check the ReconfigureRequestTime is null and LastReconfigurationTime is not null for these 2 tasks?